twisted/protocols/memcache.py

```diff
@@ -343,19 +347,19 @@ def lineReceived(self, line):
         Receive line commands from the server.
         """
         self.resetTimeout()
-        token = line.split(" ", 1)[0]
+        token = line.split(b" ", 1)[0]
         # First manage standard commands without space
-        cmd = getattr(self, "cmd_%s" % (token,), None)
+        cmd = getattr(self, u"cmd_" + token.decode("ascii"), None)
```

Comment:
this is better as cmd = getattr(self, "cmd_" + nativeString(token), None)
No need to convert to unicode here, since this is a behavior change on Python 2.